### PR TITLE
added dApp Big Ballz of Bajun desktop

### PR DESCRIPTION
**Pull request by darkfriend77**
We will switch to bbb.ajuna.io for production on the 9th July, would be awesome if it can be added for Decoded.

It's currently here ... on a solo chain https://bbb-test-524.netlify.app/

### CODE REVIEW

**Comment by leohar** [comment]
hey, thanks for your PR:

1. firstly think dapp needs adapt to mobile view, and currently user is not able to find **Login** button:

Screenshot
Screenshot

2. secondly would be nice to [integrate](https://docs.novawallet.io/nova-wallet-wiki/misc/developer-documentation/intergrate-nova-wallet) Nova  and add it as a connecting option:

Screenshot

**Comment by darkfriend77** [comment]
Okay, we will work on it.

**Comment by darkfriend77** [comment]
Okay, we added Nova Wallet as a selection and refurbished the web mobile appearance.
image

Let us know, once we can test and promote the web mobile use within Nova Wallet.

**Comment by darkfriend77** [comment]
@leohar

File: dapps/dapps_dev.json [hunk 1]
```diff
@@ -756,6 +756,16 @@
...
     },
+    {
+      "name": "Big Ballz of Bajun (Desktop mode)",
+      "icon": "https://raw.githubusercontent.com/novasamatech/nova-utils/master/icons/dapps/color/BigBallzOfBajun.png",
+      "url": "https://bbb.ajuna.io/",
+      "categories": [
+        "nft",
+        "gaming"
+      ],
+      "desktopOnly": true
```
Comment (leohar) [inline dapps/dapps_dev.json:767]: have you added this option for some purpose? cause it's opening dapp in desktop view and is implemented for dapps, that are not mobile friendly. As far as I see dapp is functioning normally in mobile view, so maybe we can remove it?

Comment (darkfriend77) [reply dapps/dapps_dev.json]: Thx, for pointing that out, yes, we set it to false. @leohar

**Review by leohar — APPROVED** [review]

**Pull request merged**